docs/source/conf.py

```diff
@@ -103,7 +103,7 @@ def get_authors_from_citation_file(file_name):
 # Change the look of autodoc classes
 # napoleon_use_ivar = True
 
-numpydoc_show_class_members = True
+numpydoc_show_class_members = False
```

Comment:
I am not sure this is having an effect, it seems that the class methods are still being listed in the documentation. Check for example the sphinx documentation of classdata_model.validate_data.DataValidator

Reply:
This is the suggested configuration do get rid of hundreds of warnings like
```
simtools/simulator.py:docstring of simulator.Simulator:110: WARNING: autosummary: stub file not found 'simulator.Simulator.simulate'. Check your autosummary_generate setting
```
(and I can't find the website anymore suggesting this, somewhere in numpydocs)